cypress/integration/gpu-dependant/canvas.js

```diff
@@ -0,0 +1,80 @@
+// Smoke test for Tectonic Explorer
+
+context('Canvas Test', () => {
+  before(function () {
+    cy.visit('/?planetWizard=true')
+    cy.wait(3000)
+  })
+
+  context('Set up same as smoke test to draw a small continent', () => {
+    it('Draws a small land mass on the planet', () => {
+      cy.get('.planet-wizard-overlay > :nth-child(1)').click()
+      // cy.get('.interaction-selector > :nth-child(3)').click()
+      cy.wait(7000)
```

Comment:
There's now a custom Cypress command `cy.waitForSpinner()` that does not depend on time (cc @kevinsantos-cc).